### ISSUE TITLE
Store last selected Group and Contact

### ISSUE THREAD

**Issue by steve-s-71** [nextcloud/contacts#376]
Proposal for an enhancement:

It would be handy if the app would store the last selected Group and Contact so that it will be displayed again if the web-site is revisited later.

**Comment by irgendwie** [comment]
Should be relatively easy to implement by using localStorage to store the gid and uid and take them into account when opening the contacts app.
For now you can simply make a 'bookmark' via your browser when the current contact is open if you really need to revisit the contact in the near future.

**Comment by skjnldsv** [comment]
Yeah, I don't like this idea. It will confuse the user if it's done in the background: "Damn it, why is this group always loading!"

We could add a bookmark button though.

**Comment by irgendwie** [comment]
I must admit I like the idea, since the app will open up where it was left. On each contactDetails update we could store the current gid and uid.
